include/net/net_pkt.h

```diff
@@ -123,6 +123,7 @@ struct net_pkt {
 
 #if defined(CONFIG_NET_L2_IEEE802154)
 	u8_t ieee802154_rssi;
+	u8_t ieee802154_lqi;
```

Comment:
There is a comment block at the beginning of this structure to remind authors that if you change this structure you need to also change the net_pkt_clone() function in net_pkt.c.

Reply:
Cloning is interesting for TCP/IP layer. I don't think anybody would need to keep track of rssi/lqi (which are valid only on rx path) beyond L2.